TheaterSchedule/Controllers/DBHelperController.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Text;
+using System.Collections.Generic;
+using System.Linq;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+
+using TheaterSchedule.Util;
+using TheaterSchedule.DAL.Contexts;
+using TheaterSchedule.DAL.Repositories;
+using TheaterSchedule.DAL.Interfaces;
+using TheaterSchedule.DAL.Entities;
+
+namespace TheaterSchedule.Controllers
+{
+    [Route("api/[controller]")]
+    public class DBHelperController : Controller
```

Comment:
Move this helper to another project that can be executed for development purposes
When main project will be started on CI there will be no need in runnung of DBHelperController